kwanj-k/ride-my-way
# Ft responsive #158483308

### What does this PR do?
Make the UI responsive to different screen sizes

### Description of Task to be completed
* Make menu collapsible when the screen is small
* Adjust the widths of different css classes to fit different screens

### How should I test it?
 * Clone this repository to a local folder
 * Navigate to the cloned repo
 * git checkout ft-home-style-#158483308
 * Open index.html using your favorite browser
* Resize the browser to different sizes to see how the different media queries perform

### What PT stories?
[ft-responsive-#158483308] https://www.pivotaltracker.com/story/show/158483308

## Screenshots

* Desktop

![oie_cxeidltte6sa](https://user-images.githubusercontent.com/33450849/41654013-b2730b12-7490-11e8-9bd7-6e791f499470.png)

* ipad
![oie_koanhcs4a7zj](https://user-images.githubusercontent.com/33450849/41654083-f49c1f4c-7490-11e8-8c0e-04e905a44873.png)

* Mobile
![oie_6yyst8whinnh](https://user-images.githubusercontent.com/33450849/41654173-4804c15c-7491-11e8-8d29-55bff887aa07.png)
